Doc/library/__main__.rst

```diff
@@ -251,9 +251,9 @@ attribute will include the package's path if imported::
     >>> asyncio.__main__.__name__
     'asyncio.__main__'
 
-This won't work for ``__main__.py`` files in the root directory of a .zip file
-though.  Hence, for consistency, minimal ``__main__.py`` like the :mod:`venv`
-one mentioned above are preferred.
+This won’t work for ``__main__.py`` files in the root directory of a
```

Comment:
A suggested change implementing the correction requested, so that anyone with commit access can apply it. (Assuming @riqts didn't disable maintainer commits.)

```suggestion
This won't work for ``__main__.py`` files in the root directory of a
```